src/components/SearchForm.tsx

```
@@ -0,0 +1,164 @@
+      navigate(`/org/${encodeURIComponent(searchConfig.owner)}${authorParam}`);
+    } else {
+      navigate(`/orgs${authorParam}`);
+    }
+  };
+
+  return (
+    <form onSubmit={handleConfigure} className="space-y-4">
+      <div className="form-control">
+        <label className="label">
+          <span className="label-text font-medium">Autor</span>
+        </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)